packages/nuxt/src/core/features.ts

```diff
@@ -1,9 +1,17 @@
 import { addDependency } from 'nypm'
 import { resolvePackageJSON } from 'pkg-types'
 import { logger } from '@nuxt/kit'
-import { isCI } from 'std-env'
+import { isCI, provider } from 'std-env'
 
-export async function ensurePackageInstalled (rootDir: string, name: string, searchPaths?: string[]) {
+const isStackblitz = provider === 'stackblitz'
+
+export async function ensurePackageInstalled (
+  rootDir: string,
+  name: string,
+  searchPaths?: string[],
+  // In StackBlitz we install packages automatically by default
+  prompt = !isStackblitz
```

Comment:
what about making this an options object in case we need to add any more in future?